gdp-src-build/conf/templates/r-car-h3-salvator-x.local.conf

```diff
@@ -14,4 +8,8 @@ include templates/renesas-rcar-gen3.local.inc
 #    comment out this line as it is not needed.
 DISTRO_FEATURES_append = " use_eva_pkg"
 
+# Select the Salvator-X board
```

Comment:
Agreed - I see now that this is better.  Looks better to have our .inc at the top whenever possible.

Reply:
My principal reason for writing the Starter Kit confs with the inc first was I knew that the block of R-Car local.conf entries in the YBSP examples were written with lazy assignment at the top and specifics at the bottom. For example "MACHINE ??=" at the top with a general default, with the specific MACHINE set later. So it made sense to do the same in the GDP templates to not be caught out with values being overwritten. That said, it looked like your reversal in the Salvator-X confs worked.

I do think that it reads marginally better with the .inc at the top. So for both reasons as I had to update the file anyway to add USB 3.0 and the H3 u-boot options I took the opportunity to change the ordering as well.